src/components/RoutingDiagram/__snapshots__/RoutingDiagram.test.tsx.snap

```diff
@@ -156,7 +156,7 @@ exports[`renders single route 1`] = `
 exports[`renders when no routes are provided 1`] = `
 <DocumentFragment>
   <div
-    class="RoutingDiagram__Wrapper-sc-o1ook0-0 ePDWDk css-vurnku"
+    class="RoutingDiagram__Wrapper-sc-i2tbb-0 ivndgC css-vurnku"
   />
 </DocumentFragment>
 `;
```

Comment:
prob didn't mean to check this in?

Reply:
this was intentional - once the package.json changes, the generated classnames change as well. 🤷